ckan/cli/search_index.py

```diff
@@ -77,6 +77,43 @@ def clear(dataset_name: str):
         clear_all()
 
 
+def list_orphans(return_list: bool=False) -> list|None:
```

Comment:
Prefer using `typing.Uinion`, because CKAN supports python >= v3.7

Reply:
Changed function to only return a list.